UITests/UITests.App/Pages/TextBoxMask.xaml

```diff
@@ -0,0 +1,52 @@
+﻿<Page
```

Comment:
Thinking we may want this in a 'Controls' sub-folder? Just trying to think of how many tests we may have in the future and organizing them a bit more now. We should keep them all in the same namespace though for searching (see other comments)

Reply:
Fixed in a34496de0201f842e5d835fa466877abcb8c448b